src/components/PhotoViewer/index.tsx

```diff
@@ -308,6 +308,7 @@ function PhotoViewer(props: Iprops) {
         });
         photoSwipe.listen('beforeChange', () => {
             const currItem = photoSwipe?.currItem as EnteFile;
+            if (!currItem) return;
```

Comment:
if we do if(!photoSwipe?currItem), will it have same effect? If yes, should we add this as first statement inside the listener?